src/gcp/storage.ts

```diff
@@ -0,0 +1,256 @@
+import * as path from "path";
```

Comment:
nit: blank line between external library imports and intra-package imports

Reply:
fixed